docs/authn.md

```diff
@@ -0,0 +1,1146 @@
+  - [Configuration](#configuration)
+  - [New Methods](#new-methods)
+  - [Transaction Model](#transaction-model)
+  - [Handling Errors](#handling-errors)
+  - [Basic Example](#basic-example)
+  - [Helpful Information and Resources](#helpful-information-and-resources)
+- [API](#api)
+  - [`signInWithCredentials(options)`](#signinwithcredentialsoptions)
+  - [`forgotPassword(options)`](#forgotpasswordoptions)
+  - [`unlockAccount(options)`](#unlockaccountoptions)
+  - [`verifyRecoveryToken(options)`](#verifyrecoverytokenoptions)
+  - [`tx.resume()`](#txresume)
+  - [`tx.exists()`](#txexists)
+  - [`session.setCookieAndRedirect(sessionToken, redirectUri)`](#sessionsetcookieandredirectsessiontoken-redirecturi)
+  - [`transaction.status`](#transactionstatus)
+    - [Common methods](#common-methods)
+      - [`cancel()`](#cancel)
+      - [`changePassword(options)`](#changepasswordoptions)
+      - [`resetPassword(options)`](#resetpasswordoptions)
+      - [`skip()`](#skip)
+    - [LOCKED_OUT](#locked_out)
+      - [`unlock(options)`](#unlockoptions)
+    - [PASSWORD_EXPIRED](#password_expired)
+    - [PASSWORD_RESET](#password_reset)
+    - [PASSWORD_WARN](#password_warn)
+    - [RECOVERY](#recovery)
+      - [`answer(options)`](#answeroptions)
+      - [`recovery(options)`](#recoveryoptions)
+    - [RECOVERY_CHALLENGE](#recovery_challenge)
+      - [`verify(options)`](#verifyoptions)
+      - [`resend()`](#resend)
+    - [MFA_ENROLL](#mfa_enroll)
+      - [`questions()`](#questions)
+      - [`enroll(options)`](#enrolloptions)
+        - [OKTA question](#okta-question)
+        - [OKTA sms](#okta-sms)
+        - [OKTA call](#okta-call)
+        - [OKTA push](#okta-push)
+        - [OKTA token:software:totp](#okta-tokensoftwaretotp)
+        - [GOOGLE token:software:totp](#google-tokensoftwaretotp)
+        - [YUBICO token:hardware](#yubico-tokenhardware)
+        - [RSA token](#rsa-token)
+        - [SYMANTEC token](#symantec-token)
+    - [MFA_ENROLL_ACTIVATE](#mfa_enroll_activate)
+      - [`resend()`](#resend-1)
+      - [`activate(options)`](#activateoptions)
+      - [`poll()`](#poll)
+      - [`prev()`](#prev)
+    - [MFA_REQUIRED](#mfa_required)
+      - [Verify Factor](#verify-factor)
+        - [OKTA question](#okta-question-1)
+        - [OKTA push](#okta-push-1)
+        - [All other factors](#all-other-factors)
+    - [MFA_CHALLENGE](#mfa_challenge)
+      - [`verify(options)`](#verifyoptions-1)
+      - [`poll(options)`](#polloptions)
+      - [`prev()`](#prev-1)
+    - [SUCCESS](#success)
+  
+## Introduction
+
+The [Okta Authentication API][] provides operations to authenticate users, perform multifactor enrollment and verification, recover forgotten passwords, and unlock accounts. It can be used as a standalone API to provide the identity layer on top of your existing application, or it can be integrated with the Okta Sessions API to obtain an Okta session cookie and access apps within Okta.
+
+## Migrating to IDX
```

Comment:
How about moving the migrating part to it's own `md` file, then link it in both `idx.md` and `authn.md`?

Reply:
My assumption is that the only people who are interested in "migrating" are authn users, so there is no reason to link from idx.md.

Reply:
I was thinking to put it in a place so it can be easily discovered. Like an existing authn user being told by the sales team to try idx, then they step directly into the idx.md. 

I might just thinking too much... just feel it probably can be clearer if it's not hidden in one of the sub docs. Probably a link at the top layer?